i18n/fr.json

```diff
@@ -3,6 +3,8 @@
   "main": {
     "activities_info": "Vous proposez un concert, une représentation, une exposition ou un film? Ou plutôt un événement sportif, une randonnée, une soirée cartes, une kermesse? Ou encore des cours, des ateliers ou des événements? Tout cela trouvera sa place dans l’UiTdatabank.",
     "activities": "activités par an",
+    "channels_info_link_text": "",
+    "channels_info_link_url": "",
```

Comment:
Are these supposed to be empty?

Reply:
Yes these aren't used on the french page, but there where warnings of missing keys when you pick french